.azure-pipelines/ci-build.yml

```diff
@@ -867,3 +871,115 @@ extends:
                         packageParentPath: "$(Pipeline.Workspace)"
                         nuGetFeedType: external
                         publishFeedCredentials: "OpenAPI Nuget Connection"
+                        
+      - stage: PushToRegistry
```

Comment:
sorry, I missed something, I believe only the job should be kept (keep the condition, and the depends on) and should be made part of the deployment stage.